src/handlers/comment/handlers/assign.ts

```diff
@@ -84,6 +86,10 @@ export const assign = async (body: string) => {
   const curDate = new Date();
   const curDateInMillisecs = curDate.getTime();
   const endDate = new Date(curDateInMillisecs + duration * 1000);
+  const deadline_msg = endDate.toUTCString();
+  const reason_msg = issue.state_reason ? issue.state_reason : "Reason";
```

Comment:
I think you should get the `reason` from the `wallets` table.

Reply:
Resolves comment
https://github.com/starlitnightsky/Real-Estate/issues/10#issuecomment-1598049354
Default Reason is null, so there is no reason at first, I think.